### ISSUE TITLE
Query ApplicationInisights - proper library

### ISSUE THREAD

**Issue by SteppingRazor** [Azure/azure-sdk-for-net#14126]
**Query/Question**
Hello, I'm a little bit confused about Azure libraries. I want to query the application insights (using kusto) from .net application. 

Right now I'm using Microsoft.Azure.ApplicationInsights library. This works fine, but what bothers me is this library latest version is 0.9.0-preview and it is from April 2018.

Is there a newer version of this library? Did they migrate somewhere? If no, is there any library that I can use which will provide me with the same functionality (authentication with the token and query the logs of application insights)? I cannot find any information about it on the Web.
_

**Environment:**
 - Microsoft.Azure.ApplicationInsights 0.9.0-preview
 - .NET Core SDK Version:   3.1.302
 - IDE and version: Visual Studio 16.6.4

**Comment by jsquire** [comment]
Thank you for your feedback.  Tagging and routing to the team best able to assist.

**Comment by SteppingRazor** [comment]
@jsquire I won't get any answer from these guys, won't I?

**Comment by jsquire** [comment]
@SteppingRazor:  I'm sorry that you haven't gotten one yet, and I sincerely hope that we can get you one.

@AlexGhiondea, @lilyjma:  Do either of you have a contact that you could help route to,; it seems the service team alias isn't responding.

**Comment by jsquire** [comment]
@lmolkova: Would you be able to help route to the best point of contact?

**Comment by lilyjma** [comment]
@reyang : Could you please take a look? Thanks!

**Comment by reyang** [comment]
@lilyjma I have **no** idea what this SDK is. My suggestion is to find the source code and use `git blame` to find out the owner.

**Comment by chidozieononiwu** [comment]
Tagging possible owners
@frankguodongchen @alexeldeib @dsgouda

**Comment by alexeldeib** [comment]
@alongafni fyi

I worked on this SDK a while back, I've since changed teams. I have a sync with the owning team next week. I'm not really sure what to say about the current SDK process as I've been out of the loop for over a year -- I'd love to put them in contact with someone from the Azure SDK team who can guide them in the right direction on how they should move forward maintaining a data plane SDK.

any volunteers to be a point of contact // join our chat? 🙂 Feel free to ping me internally, my alias is aleldeib
